build/Makefile

```diff
@@ -70,14 +70,9 @@ build-registers:	$(REGISTERS_ADOC)
 	../registers.py --adoc $@ --adoc-definitions $(patsubst %.adoc,%-def.adoc,$@) $<
 
 debug_defines: debug_defines.h debug_defines.c $(patsubst %,../xml/%,$(REGISTERS_ADOC:.adoc=.xml)) ../registers.py
-	../registers.py $@ --cgetters $(filter %.xml, $^)
-
-debug_defines.%:
```

Comment:
By removing this make's target `debug_defines.%` the build `make debug_defines` fails on clean build, e.g. after a clean `git clone`
You need to drop `debug_defines.h` and `debug_defines.c` at line 74

Reply:
Good point. Fixed.